### ISSUE TITLE
tracker roll volume too high

### ISSUE THREAD

**Issue by sinusoidaltendencies** [polyend/TrackerBetaTesting#1552]
I've noticed that the output volume of steps with rolls on them (no matter what the type and value) are generally much louder than the rest of the steps in the sequence. Especially if there are empty steps after the the step with the roll. A work around is to set the second effect slot to Velocity and turn it down but if the second effect slot is already being used for something else like Micro move or Chance etc... then you have no control over roll volume. 

I wonder if it would be possible to have a customizable "global roll volume" menu somewhere within the tracker architecture. 

It's such a sweet effect but can be a bit unwieldy if unrefined.

**Comment by m27underground** [comment]
yes, i agree that roll volume has been an issue that can become annoying to work around when a global option is all it needs.
